src/errors.rs

```diff
@@ -39,6 +39,9 @@ pub enum OrandaError {
     #[error(transparent)]
     UrlParse(#[from] url::ParseError),
 
+    #[error(transparent)]
+    GenerateCss(#[from] oranda_generate_css::errors::GenerateCssError),
+
```

Comment:
```suggestion
    #[error(transparent)]
    #[diagnostic(transparent)]
    GenerateCss(#[from] oranda_generate_css::errors::GenerateCssError),

```

Lets miette know that the inner error is also a miette error, prints inner details right.